src/uberjob/_registry.py

```diff
@@ -100,27 +100,27 @@ def get(self, node: Node) -> typing.Optional[ValueStore]:
         v = self.mapping.get(node)
         return v.value_store if v else None
 
-    def keys(self) -> typing.Iterable[Node]:
+    def keys(self) -> typing.KeysView[Node]:
         """
         Get all registered :class:`~uberjob.graph.Node` instances.
 
-        :return: An iterable of :class:`~uberjob.graph.Node`.
+        :return: A keys view of :class:`~uberjob.graph.Node`.
         """
         return self.mapping.keys()
 
-    def values(self) -> typing.Iterable[ValueStore]:
+    def values(self) -> typing.List[ValueStore]:
```

Comment:
Because of the projection done to each value, it's not easy to make a true ValuesView here. We were already returning a list but advertising only Iterable. Updating to specifying a list.